lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -36,15 +37,49 @@ def string_to_array(string, delimiter = /[,\n]/)
         string.strip.split(delimiter).map(&:strip)
       end
 
+      def app_id_from_params(params)
+        plist_path = params[:googleservice_info_plist_path]
+        if params[:app]
+          app_id = params[:app]
+        elsif xcode_archive_path
+          app_id = get_ios_app_id_from_archive_plist(xcode_archive_path, plist_path)
+        elsif plist_path
+          app_id = get_ios_app_id_from_plist(plist_path)
```

Comment:
probably add some logging here indicating that the app ID was taken directly from the plist_path specified in the parameters

Reply:
Done